plugins/module_utils/cloudstack.py

```diff
@@ -35,6 +35,7 @@ def cs_argument_spec():
         api_http_method=dict(choices=['get', 'post'], default=os.environ.get('CLOUDSTACK_METHOD')),
         api_timeout=dict(type='int', default=os.environ.get('CLOUDSTACK_TIMEOUT')),
         api_region=dict(default=os.environ.get('CLOUDSTACK_REGION') or 'cloudstack'),
+        api_ssl_verify=dict(default=os.environ.get('CLOUDSTACK_VERIFY')),
```

Comment:
hmm `ssl_verify` is IMHO misleading: mind changing to `api_verify_ssl_cert`?

Reply:
OK, done.